src/emulator/auth/operations.ts

```diff
@@ -3025,36 +3025,49 @@ async function fetchBlockingFunction(
   }, timeoutMs);
 
   let response;
+  let res;
```

Comment:
nit: we just need to extact `ok`, no need for the whole `res`. Also, please type these variables.

Reply:
Oh nvm, I see your comment above

Reply:
Haha, already applied the feedback, all good